buildSrc/subprojects/kotlin-dsl/src/main/kotlin/plugins/kotlin-dsl-plugin-bundle.gradle.kts

```diff
@@ -42,31 +41,21 @@ afterEvaluate {
     }
 }
 
-
-tasks {
-    validatePlugins {
```

Comment:
This was a duplication. This is configured for all "java/kotlin" projects already.